pages/history/index.jsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react'
+
+import styles from './history.module.scss'
+
+import { Markdown, Page } from 'components'
+import historyData from 'data/history.yml'
+import retrieveContent from 'content'
+
+const LinkCard = ({ link }) => (
+  <div>
+    <h4>
+      <a href={link.href} target="_blank" rel="noreferrer">
+        {link.caption}
+      </a>
+    </h4>
+    <p>{link.source}</p>
+  </div>
+)
+
+const Milestone = ({ milestone }) => (
```

Comment:
Move this to a separate component and create its own style sheet for this. It also applies to the `LinkCard`.

If this is a high-level component, i.e. not a pure HTML element repeating, it accepts `data`, `tag` and the rest of props are spread as HTML props:

```suggestion
const Milestone = ({ data, tag: Tag = 'div', ...htmlProps }) => (
```

where:

- `...htmlProps` enable you passing everything you need, e.g. `className`, `aria-labelledby`, `aria-hidden` and so on, any attributes that you might want to add to support accessibility, other HTML feature or whatever
- `tag` enables you passing any semantic element name or yet another component giving some flexibility for reuse
- `data` is derived from `data-*` attributes, i.e. you pass an object that could be serialised as `data-title`, `data-image-src`, `data-id` etc preserving full compatibility with the valid HTML.